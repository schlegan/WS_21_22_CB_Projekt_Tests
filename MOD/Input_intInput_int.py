print(input_int() % input_int()) # 6
print(input_int() % input_int()) # 0
print(input_int() % input_int())# -6
print(input_int() % input_int())# 6
print(input_int() % input_int()) # -6
print(input_int() % input_int()) # 0
# Jetz nochmal alles fÃ¼r maximal groÃŸe Werte.
print(input_int() % input_int()) # 9223372036854775807
print(input_int() % input_int()) # -1
print(input_int() % input_int()) # 0
print(input_int() % input_int()) # 0
