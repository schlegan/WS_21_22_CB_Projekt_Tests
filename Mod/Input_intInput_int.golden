60-66-609223372036854775807-100
